src/tests/Interop/common/xplatform.h

```diff
@@ -8,6 +8,104 @@
 
 #ifndef WINDOWS
 
+#define __RPC_FAR
+#define DECLSPEC_UUID(x)
+#define DECLSPEC_NOVTABLE
+#define MIDL_INTERFACE(x)   struct DECLSPEC_UUID(x) DECLSPEC_NOVTABLE
+#define interface struct
```

Comment:
The build failures on macOS are from this define for `interface`. This file is also used by Objective-C test sources - those tests have `OBJC_TESTS` defined, so you should be able to use that to avoid the conflicting define here.

Reply:
ok